src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java

```diff
@@ -35,8 +35,21 @@ public static void main(String[] args) {
      * </ul>
      */
     static String getGreetingByHour(int hour) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        final int hour1 = 6;
+        final int hour2 = 12;
+        final int hour3 = 18;
+        final int hour4 = 24;
+        final int hour5 = 0;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"